jwt/api_jwt.py

```diff
@@ -161,8 +158,17 @@ def _validate_aud(self, payload, audience):
             raise InvalidAudienceError('Invalid claim format in token')
         if any(not isinstance(c, string_types) for c in audience_claims):
             raise InvalidAudienceError('Invalid claim format in token')
-        if audience not in audience_claims:
-            raise InvalidAudienceError('Invalid audience')
+
+        if isinstance(audience, string_types):
+            audience = [audience]
+
+        if not isinstance(audience, list):
+            raise InvalidAudienceError('Invalid audience format')
+
+        for aud in audience:
+            if aud in audience_claims:
```

Comment:
Since we are now checking multiple values to see if they are in `audience_claims` do you think it might make more sense to make `audience_claims` a set?